app/src/main/res/layout/activity_main.xml

```diff
@@ -5,15 +5,164 @@
+        app:layout_constraintTop_toTopOf="@id/background_my_profile"
+        app:layout_constraintVertical_bias="0.067"
+        app:strokeColor="@color/log_out"
+        app:strokeWidth="2dp" />
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/view_my_contacts"
+        android:layout_width="0dp"
+        android:layout_height="55dp"
+        style="@style/ButtonStyle.Contacts"
+        android:text="@string/view_my_contacts"
+        app:cornerRadius="6dp"
+        android:textStyle="bold"
         app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintTop_toBottomOf="@id/background_my_profile"
+        app:layout_constraintVertical_bias="0.95" />
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/edit_profile"
+        android:layout_width="0dp"
+        android:layout_height="40dp"
+        android:minHeight="48dp"
+        style="@style/ButtonStyle.EditProfile"
+        android:text="@string/edit_profile"
+        app:cornerRadius="6dp"
+        app:layout_constraintBottom_toTopOf="@id/view_my_contacts"
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintTop_toBottomOf="@id/background_my_profile"
+        app:layout_constraintVertical_bias="0.95"
+        app:strokeColor="@color/black"
+        app:strokeWidth="2dp" />
+
+    <LinearLayout
+        android:id="@+id/links"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="center"
+        android:orientation="horizontal"
+        app:layout_constraintBottom_toTopOf="@id/edit_profile"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toBottomOf="@id/background_my_profile">
+
+        <ImageButton
+            android:id="@+id/imageButtonFacebook"
+            style="@style/ImageButtonStyle"
+            app:srcCompat="@drawable/facebook"
+            android:contentDescription="@string/facebook_description"/>
+
+        <ImageButton
+            android:id="@+id/imageButtonLinkedIn"
+            style="@style/ImageButtonStyle"
+            app:srcCompat="@drawable/linkedin"
+            android:contentDescription="@string/linkedin_description"/>
+
+        <ImageButton
+            android:id="@+id/imageButtonInstagram"
+            style="@style/ImageButtonStyle"
+            app:srcCompat="@drawable/instagram"
+            android:contentDescription="@string/instagram_description" />
+
+    </LinearLayout>
+
+    <de.hdodenhof.circleimageview.CircleImageView
+        android:id="@+id/icon_profile"
+        android:layout_width="113dp"
```

Comment:
Try to replace your approach with [Ratio](https://developer.android.com/reference/androidx/constraintlayout/widget/ConstraintLayout#dimensions-constraints).

Reply:
done

Reply:
I was trying to say that using dimensionratio and other specific attributes, you can set width of image to 33% of screen width and make that image square, so height will be equal to width. Try to figure it out and do it.

Reply:
done